### PR TITLE
[DOCS] Use https link

### PR DESCRIPTION
**Pull request by tqchen**

### CODE REVIEW

**Comment by tqchen** [comment]
cc @yzhliu从

File: docs/contribute/document.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@
 Write Document and Tutorials
 ============================
 
-We use the `Sphinx <http://sphinx-doc.org>`_ for the main documentation.
+We use the `Sphinx <https://sphinx-doc.org>`_ for the main documentation.
```
Comment (yzhliu) [inline docs/contribute/document.rst:23]: this one seems to be not working.

Comment (tqchen) [reply docs/contribute/document.rst]: good catch, just reverted this one to http

**Comment by tqchen** [comment]
ping @yzhliu

**Comment by yzhliu** [comment]
Thanks @tqchen

**Pull request merged**